db/Models/news.models.js

```diff
@@ -94,25 +94,61 @@ const insertCommentByArticle = (article_id, body) => {
       }
 
       const commentValues = Object.values(body);
-      console.log(commentValues);
+      // console.log(commentValues);
```

Comment:
remember to remove console logs & code comments once you're done with them to avoid cluttering main